WordPress/Classes/Utility/ImmuTable.swift

```diff
@@ -279,6 +279,14 @@ public class ImmuTableViewHandler: NSObject, UITableViewDataSource, UITableViewD
         return cell
     }
 
+    public func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return viewModel.sections[section].headerText
+    }
+
+    public func tableView(tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        return viewModel.sections[section].footerText
```

Comment:
Subclasses again! :scream: 
Yes, we probably want custom appearance. No, I don't want to have to use a custom view subclass just to display some text. But I might have to. I'll look into it a bit more.